src/testing/benchmark.go

```diff
@@ -436,8 +444,9 @@ func (b *B) Run(name string, f func(b *B)) bool {
 	b.hasSub = true
 	sub := &B{
 		common: common{
-			name:  benchName,
-			level: b.level + 1,
+			output: &logger{},
```

Comment:
this is not something I care about but some people worry when fields are not in alphaetical order ...

Reply:
In this case, `output` is actually the first field in `common`.